samples/react-application-collab-footer/README.md

```diff
@@ -1,52 +1,74 @@
 # Collaboration Footer Application Customizers
 
-> NOTE: This sample is available in the [PnP Starter Kit](https://github.com/pnp/sp-starter-kit). You can find the sample code for this extension at https://github.com/pnp/sp-starter-kit/tree/master/source/react-application-collab-footer.
+This application customizer provides you the ability to include a footer designed for normal and group associated teams sites. The footer includes sets of links configured within a taxonomy term set via the term store. A second set of links are personalized links, unqiue to each user, stored within each user's user profile within a user profile property. If this property does not exist in the user profile service, this capability will be hidden. 
 
-This application customizer provides you the ability to include a footer designed for normal and group associated teams sites. The footer includes sets of links configured using the tenant wide deployment list at the app catalog. A second set of links are personalized links, unqiue to each user, stored within each user's user profile within a user profile property. If this property does not exist in the user profile service, this capability will be hidden. 
+![Collaboration Footer](../../assets/images/components/ext-collab-footer.gif)
```

Comment:
seems we should also move the assets image from starter kit and update the path here so that it will be also present here